char/liberate/Makefile.template

```diff
@@ -0,0 +1,96 @@
+VPATH:=make
```

Comment:
do we really need a template here, couldn't those be passed as arguments with `make VARIANT=12T`?

Reply:
agreed..deleted.